pycontrails/core/models.py

```diff
@@ -311,14 +324,18 @@ def eval(self, source: Any = None, **params: Any) -> ModelOutput:
     def interp_kwargs(self) -> dict[str, Any]:
         """Shortcut to create interpolation arguments from :attr:`params`.
 
-        Creates the required dict format for ``RegularGridInterpolator``
-        attached to ``MetDataArray``.
-
         Returns
         -------
         dict[str, Any]
-            Dictionary with keys "method", "bounds_error", and "fill_value" as determined by
-            :attr:`params`.
+            Dictionary with keys
+
+            - "method"
+            - "bounds_error"
+            - "fill_value"
+            - "localize"
+            - "use_indices"
+
+            as determined by :attr:`params`.
         """
         return {
```

Comment:
It feels like this should contain the `interpolation_q_method` and then you could remove the `interpolation_` prefix?

Reply:
fe10a1df11aed5cedc1afe767701ef1a869b7059